### PR TITLE
Upstream 2.36.4

### LINKED ISSUE #5: 2.36.4 fixes CVE-2022-41903 and CVE-2022-23521
https://github.blog/2023-01-17-git-security-vulnerabilities-announced-2/

https://lwn.net/Articles/920256/

### PR DESCRIPTION
**Pull request by dabio**
Fixes #5

### CODE REVIEW

File: git236.spec [hunk 1]
```diff
@@ -1075,6 +1075,10 @@ rmdir --ignore-fail-on-non-empty "$testdir"
 %{?with_docs:%{_pkgdocdir}/git-svn.html}
 
 %changelog
+* Tue Jan 20 2023 Danilo Braband <dbraband@gmail.com> - 2.36.4-1
```
Comment (jeffsheltren) [inline git236.spec]: Can you fix this date please? Either the day is off or the date is off.

**Review by jeffsheltren — CHANGES_REQUESTED** [review]
Thanks! Can you fix the changelog date? Then I can merge this.

File: git236.spec (hunk 1 above)
Comment (dabio) [reply git236.spec]: Fixed it. Sorry.

**Review by jeffsheltren — APPROVED** [review]

**Comment by jeffsheltren** [comment]
I'll get this pushed to IUS Testing as soon as I can.

**Comment by carlwgeorge** [comment]
I've pushed git236-2.36.3-1.el7.ius to the testing repo.

**Comment by rtkennedy** [comment]
Should it be git236-2.36.4-1 being pushed to the testing repo instead of git236-2.36.3-1?

**Comment by jeffsheltren** [comment]
Yeah, there was an issue with the CI build. I've rebuilt and pushed git236-2.36.4-1 to IUS testing.

**Pull request merged**